.github/workflows/main.yml

```diff
@@ -3,21 +3,18 @@ name: Main tests
 on: push
 
 jobs:
-  kahlan:
+  tests:
     runs-on: ubuntu-20.04
```

Comment:
This is a different version to the Dockerfile, but maybe it's changed later in the history?